Stellar-ledger.x

```diff
@@ -130,6 +130,11 @@ enum LedgerUpgradeType
     LEDGER_UPGRADE_CONFIG = 6
 };
 
+struct ConfigUpgradeSetKey {
+    Hash contractID;
+    Hash contentHash;
+};
+
```

Comment:
If my understanding correctly, the `ConfigUpgradeSetKey` here must be a contract data belong to `ContractID`? Does that mean only a ContractData entry can be upgraded? 
I'm thinking in the context of network-wide setting such as metering cost params that wouldn't belong to a contract, how will it work?